addons/web_editor/static/src/js/backend.js

```diff
@@ -351,7 +351,12 @@ var FieldTextHtml = widget.extend({
         });
     },
     render_value: function() {
-        if (this.lang !== this.view.dataset.context.lang || this.$iframe.attr('src').match(/[?&]edit_translations=1/)) {
+        if (this.$iframe.attr('src').match(/[?&]edit_translations=1/)) {
+            return;
+        }
+        // ONLY HAVE THIS IN 8.0 AND SAAS-6
```

Comment:
I think you missed the version. 8.0 is a complete mess, but this issue happened in 9.0+